### PR TITLE
Improve connectivity report output headers

### PR DESCRIPTION
**Pull request by olasaadi99**

### CODE REVIEW

File: pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Connectivity for VPC test-vpc1-ky
+# Endpoint connectivity for VPC test-vpc1-ky
 ## Endpoint connectivity report
```
Comment (zivnevo) [inline pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md:2]: Can we avoid this duplication as well?

File: pkg/ibmvpc/examples/out/analysis_out/acl_testing5_all_vpcs_subnetsBased_withPGW.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Connectivity for VPC test-vpc-ky1
+# Subnet connectivity for VPC test-vpc-ky1
 ## Subnets connectivity report
```
Comment (zivnevo) [inline pkg/ibmvpc/examples/out/analysis_out/acl_testing5_all_vpcs_subnetsBased_withPGW.md:2]: here too

File: pkg/ibmvpc/examples/out/analysis_out/tgw_larger_example_all_vpcs_.txt [hunk 3]
```diff
@@ -37,24 +37,24 @@ vsi3a-ky[10.240.3.5] => vsi3b-ky[10.240.3.4] : All Connections
...
 
-Connectivity for VPC test-vpc3-ky
+Endpoint connectivity for VPC test-vpc3-ky
 Public Internet (all ranges) => vsi32-ky[10.240.128.4] : All Connections
 vsi31-ky[10.240.31.4] => vsi32-ky[10.240.128.4] : All Connections
 vsi32-ky[10.240.128.4] => Public Internet (all ranges) : All Connections
 vsi32-ky[10.240.128.4] => vsi31-ky[10.240.31.4] : All Connections
 
-Connectivity for VPC zn-vpc1
+Endpoint connectivity for VPC zn-vpc1
```
Comment (zivnevo) [inline pkg/ibmvpc/examples/out/analysis_out/tgw_larger_example_all_vpcs_.txt:54]: Can we have something like
```suggestion
Endpoint connectivity for VPC zn-vpc1
<nothing to report>
```

File: pkg/ibmvpc/examples/out/analysis_out/tgw_larger_example_all_vpcs_subnetsBased_withPGW.txt [hunk 4]
```diff
@@ -12,37 +11,30 @@ subnet31-ky => subnet21-ky : All Connections
...
 subnet32-ky => subnet31-ky : All Connections
 
-Connectivity for VPC zn-vpc1
-combined connections between subnets:
+Subnet connectivity for VPC zn-vpc1
 
 
-Connectivity for VPC zn-vpc2
-combined connections between subnets:
+Subnet connectivity for VPC zn-vpc2
```
Comment (zivnevo) [inline pkg/ibmvpc/examples/out/analysis_out/tgw_larger_example_all_vpcs_subnetsBased_withPGW.txt:37]: Here too
```suggestion
Subnet connectivity for VPC zn-vpc2
<nothing to report>
```

**Review by zivnevo — CHANGES_REQUESTED** [review]

File: pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md (hunk 1 above)
Comment (olasaadi99) [reply pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md:2]: sure, do you want to leave first line or sec line?

Comment (zivnevo) [reply pkg/ibmvpc/examples/out/analysis_out/acl_testing3_all_vpcs_.md:2]: First line

**Review by zivnevo — APPROVED** [review]
Thanks! Looks great.

**Pull request merged**